.storybook/static/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
could this be moved to the parent folder so that the `static` isn't committed?

Reply:
added this so that the `static` folder would get committed, the build fails if it does not find it

https://app.circleci.com/pipelines/github/Financial-Times/x-dash/3148/workflows/9fe4b5f4-8e03-4a10-9422-e4f1521152a7/jobs/6446